### PR TITLE
Create Architecture folder

### PR DESCRIPTION
**Pull request by pxydn**

### CODE REVIEW

**Comment by marbiano** [comment]
@pxydn is the drawing up somewhere for me to review?

File: Architecture/Readme.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+4) Code: Zoom in again, and this time you model every element from 3rd level using standard modeling techniques (mostly with will be UML, ERD, Sequence diagrams etc.)
+
+Source: 
+Website: https://c4model.com/
+Youtube tutorial by the author: https://www.youtube.com/watch?v=x2-rSnhpw0g&t 
+His book: 
+
+<h1>Drawing the diagrams and collaborating</h1>
+
+I suggest using draw.io, as Cryptopanter is already familirizing himself with it and Inow already started a concept.
```
Comment (LNow) [inline Architecture/Readme.md:21]:
```suggestion
I suggest using draw.io, as Cryptopanter is already familirizing himself with it and LNow already started a concept. 
```

File: Architecture/Readme.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+<h1>Architecture model</h1>
+We will use the C4model to describe the DAOos architecture. I think (314159...) this is a candidate for a standard way of working within Labs3. 
+
+In short (thanks Inow) the C4 model is a way of visualizing software using 4 levels of details:
```
Comment (LNow) [inline Architecture/Readme.md:4]:
```suggestion
In short (thanks LNow) the C4 model is a way of visualizing software using 4 levels of details:
```
Typo

**Comment by 314159265359879** [comment]
> 
> 
> @pxydn is the drawing up somewhere for me to review?

No change from the one cited on our Discord here: https://discord.com/channels/621759717756370964/829029866477060146/834104330775822379

We mostly hashed out how we want to work on this.

**Review by LNow — APPROVED** [review]
LGTM

**Review by 314159265359879 — APPROVED** [review]
Later we can also add a .png from draw.io as a export to this folder for easy viewing on github.

**Review by JMKJR — CHANGES_REQUESTED** [review]
I believe Architecture and coding standards files should go in a `docs` folder in top-level directory next to the README.md. This will keep the top-level from getting cluttered with docs. Also `Architecture` folder inside `docs` should be lowercase `architecture` imo to keep with the existing convention in our project structure :)

**Comment by JMKJR** [comment]
This works for me! I'd like @LNow and @314159265359879 to re-review before I approve to avoid an auto-merge before they see the latest changes.

**Comment by LNow** [comment]
Still green light from me :+1:

**Review by JMKJR — APPROVED** [review]

**Pull request merged**